crates/task-impls/src/consensus.rs

```diff
@@ -1331,7 +1331,7 @@ where
                 .sign_validating_or_commitment_proposal::<I>(&leaf.commit());
             // TODO: DA cert is sent as part of the proposal here, we should split this out so we don't have to wait for it.
             let proposal = QuorumProposal {
-                block_header: TYPES::BlockHeader::new(*payload_commitment, &parent_header),
+                block_header: TYPES::BlockHeader::new(*payload_commitment, (), &parent_header),
```

Comment:
Same here, but actually probably better to do `leaf.block_header.clone()` rather than rebuild the header from scratch